picard/ui/mainwindow.py

```diff
@@ -501,9 +501,13 @@ def create_actions(self):
 
         self.view_log_action = QtWidgets.QAction(_("View Error/Debug &Log"), self)
         self.view_log_action.triggered.connect(self.show_log)
+        # TR: Keyboard shortcut for "View Error/Debug Log"
+        self.view_log_action.setShortcut(QtGui.QKeySequence(_("Ctrl+M")))
```

Comment:
You would want this to be `Ctrl+L` to correspond to the type hint in the action widget above.

Reply:
CTRL-L is for keyboard shortcut for lookup

Reply:
You might want to change the key hint then. Is ctrl e and d also taken?

Reply:
Both are free . Which one should i use?

Reply:
Sorry about ctrl-d it's used for opening folder